server/data/incentivesApi.ts

```diff
@@ -25,6 +25,11 @@ export interface PrisonIncentiveLevel {
   privilegedVisitOrders: number
 }
 
+export type PrisonIncentiveLevelUpdate = Omit<
```

Comment:
I do wonder if for these `*Update` types `Pick<Type, Keys>` would be a better alternative to
`Omit<>`.

The idea is that we have a list of fields you can update rather than get all the list and remove some. I know the `PrisonIncentiveLevel` type is probably not going to change much but in theory if a new property is added it would be allowed inside `PrisonIncentiveLevelUpdate`

Although I appreciate this is a bit academic as we the API would respond restrict things (or ignored), etc, so don't worry too much about this. It's just a thought ☺️

Reply:
hmmmm i thought the opposite weirdly: to omit the read-only / irrelevant fields so that future additions are automatically included in the update.

also: omit 3 fields vs pick 7… more to write in the reversed version!